content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -637,7 +637,7 @@ either do not manually edit the role, or disable auto-reconciliation.
 </tr>
 <tr>
 <td><b>system:public-info-viewer</b></td>
-<td><b>system:authenticated</b> and <b>system:unauthenticated</b> groups</td>
+<td><b>system:authenticated</b> and <b>system:unauthenticated</b> group</td>
```

Comment:
I think the current text is correct, since this binds to two groups. This is not corresponding to the `Group` kind in the API.

Reply:
Thanks, @liggitt  for this information.